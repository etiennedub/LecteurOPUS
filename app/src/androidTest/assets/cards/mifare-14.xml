<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">044727EC</List><List index="1">BA334980</List><List index="2">4048E000</List><List index="3">00000FFF</List><List index="4">00010001</List><List index="5">30DF8523</List><List index="6">00000000</List><List index="7">188B797F</List><List index="8">01C6577C</List><List index="9">77C10308</List><List index="10">67008000</List><List index="11">20280526</List><List index="12">01C65A10</List><List index="13">A106051F</List><List index="14">42000000</List><List index="15">2028A2BC</List></Pages></Mifare>