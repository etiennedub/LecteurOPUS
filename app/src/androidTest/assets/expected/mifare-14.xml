<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="26/11/2016" Time="21:28" Transfert="false" OperatorId="20" BusId="96" OperatorName="RTC" BusName="METROBUS 801" /><List index="1" Date="26/11/2016" Time="15:58" Transfert="false" OperatorId="20" BusId="16" OperatorName="RTC" BusName="LEBUS 28" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>